### PR TITLE
test(Button): test for Button added

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/components/Button/utils.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import {ButtonSize, ButtonView} from '@gravity-ui/uikit';
 
+export const ICON_QA = 'button-icon';
```
Comment (yuberdysheva) [inline src/components/Button/utils.ts:3]: Why is ICON_QA here? Maybe it should be replaced to Button.test.tsx?

File: src/components/Button/__tests__/Button.test.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import Button from '../Button';
+
+const qaId = 'button-component';
+
+const buttonProps = {
+    text: 'Button Text',
+    url: 'https://github.com/gravity-ui/',
+    target: '_blank',
+    img: {
+        url: 'https://storage.yandexcloud.net/cloud-www-assets/constructor/storybook/images/img-gray.png',
```
Comment (yuberdysheva) [inline src/components/Button/__tests__/Button.test.tsx:18]: it is image for the button? Why is it not icon?

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/components/Button/utils.ts (hunk 1 above)
Comment (niktverd) [reply src/components/Button/utils.ts:3]: It is used in both test and component files. Let him be here )

File: src/components/Button/__tests__/Button.test.tsx (hunk 2 above)
Comment (niktverd) [reply src/components/Button/__tests__/Button.test.tsx]: Sure

**Review by yuberdysheva — APPROVED** [review]

**Pull request merged**